admin/client/components/ItemsTable.js

```diff
@@ -1,20 +1,149 @@
+const classnames = require('classnames');
+
 const Columns = require('../columns');
 const CurrentListStore = require('../stores/CurrentListStore');
 const ListControl = require('./ListControl');
 const React = require('react');
+const ReactDOM = require('react-dom');
+const { DragDropContext, DropTarget, DragSource } = require('react-dnd');
+const HTML5Backend = require('react-dnd-html5-backend');
+
 
 const CONTROL_COLUMN_WIDTH = 26;  // icon + padding
 
-var ItemsTable = React.createClass({
+/**
+ * Implements drag source.
+ */
+const itemSource = {
+	beginDrag(props) {
```

Comment:
function names and argument definitions should have a space between them, i.e

`beginDrag (props) {`